### ISSUE TITLE
Tooltip content not focusable using keyboard

### ISSUE THREAD

**Issue by ev-codes** [bpmn-io/properties-panel#493]
### Describe the Bug

When a tooltip is placed on a group, its contents are not focusable using the keyboard. This prevents keyboard navigators from opening links inside tooltip content (for example, documentation links).

Tooltips on other components (like `Textfield`) work as expected.

### Steps to Reproduce

1. Navigate to a group with a tooltip that contains a link (e.g., in `camunda-modeler`, click on a service task and navigate to Task definition)
2. The tooltip should open automatically

The user is unable to focus on the link inside using only the keyboard.

### Expected Behavior

The user is able to focus (and open) the link using only the keyboard, like `Textfield` allows.

### Environment

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: macOS 26
- Library version: 3.40.6

**Comment by AlekseyManetov** [comment]
This broke the Web Modeler case, where the tooltip is now covered by the Side Panel buttons:

<img width="1093" height="432" alt="Image" src="https://github.com/user-attachments/assets/8f9d2f58-50f5-40b1-97d6-4840ed0db3ba" />

This was probably caused by us removing the portaling; the tooltip now lives inside the bio-properties-panel-group-header element's stacking context, which has a `zIndex: 10`. This is lower than the button's `zIndex: 50`.

**Comment by AlekseyManetov** [comment]
> This broke the Web Modeler case, where the tooltip is now covered by the Side Panel buttons:
> 
> <img alt="Image" width="1093" height="432" src="https://private-user-images.githubusercontent.com/14283670/598149049-8f9d2f58-50f5-40b1-97d6-4840ed0db3ba.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Nzk4ODQ3NjYsIm5iZiI6MTc3OTg4NDQ2NiwicGF0aCI6Ii8xNDI4MzY3MC81OTgxNDkwNDktOGY5ZDJmNTgtNTBmNS00MGIxLTk3ZDYtNDg0MGVkMGRiM2JhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNTI3VDEyMjEwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRlYWJmN2NlYTBhZDUxM2E2ZjRiMmFkN2Y3MmExZWMyOTUzZmRiODgxODczNGIxMmYwN2YyYjM4NDk3YjMxYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.K4dQsO7pxRPkks00QDMrmsXBwzRVxL7Wi2suhQflDVM">
> This was probably caused by us removing the portaling; the tooltip now lives inside the bio-properties-panel-group-header element's stacking context, which has a `zIndex: 10`. This is lower than the button's `zIndex: 50`.

This is fixed by https://github.com/camunda/camunda-hub/pull/24636. Here is also alernative approach to fix this issue without portal removing - https://github.com/bpmn-io/properties-panel/pull/501
